LibRangeCheck-2.0/LibRangeCheck-2.0.lua

```diff
@@ -1360,35 +1447,40 @@ function lib:activate()
     if not self.frame then
         local frame = CreateFrame("Frame")
         self.frame = frame
+
         frame:RegisterEvent("LEARNED_SPELL_IN_TAB")
         frame:RegisterEvent("CHARACTER_POINTS_CHANGED")
-        if not IsClassic and not IsTBC then
+        frame:RegisterEvent("SPELLS_CHANGED")
+
+        if isRetail then
             frame:RegisterEvent("PLAYER_TALENT_UPDATE")
         end
-        frame:RegisterEvent("SPELLS_CHANGED")
+
         local _, playerClass = UnitClass("player")
         if playerClass == "MAGE" or playerClass == "SHAMAN" then
             -- Mage and Shaman gladiator gloves modify spell ranges
             frame:RegisterUnitEvent("UNIT_INVENTORY_CHANGED", "player")
         end
     end
+
     initItemRequests()
-    self.frame:SetScript("OnEvent", function(frame, ...) self:OnEvent(...) end)
-    self.frame:SetScript("OnUpdate", function(frame, elapsed)
+
+    self.frame:SetScript("OnEvent", function(_, ...) self:OnEvent(...) end)
+    self.frame:SetScript("OnUpdate", function(_, elapsed)
         lastUpdate = lastUpdate + elapsed
         if lastUpdate < UpdateDelay then
             return
         end
         lastUpdate = 0
         self:initialOnUpdate()
     end)
+
     self:scheduleInit()
 end
 
 --- BEGIN CallbackHandler stuff
 
 do
-    local lib = lib -- to keep a ref even though later we nil lib
```

Comment:
I don't really get what this was for _(or the one below it)_ the lib was already localized